devbin/dev_proxy.py

```diff
@@ -50,12 +50,15 @@ async def render_html(request: web.Request, context: dict):
     return await render_template(SERVICE, request, **context, cookie_domain='localhost:8000')
 
 
+BC = web.AppKey('backend_client', Session)
+
+
 async def on_startup(app: web.Application):
-    app['backend_client'] = Session(credentials=hail_credentials())
+    app[BC] = Session(credentials=hail_credentials())
 
 
 async def on_cleanup(app: web.Application):
-    await app['backend_client'].close()
+    await app[AppKeys.BC].close()
```

Comment:
```suggestion
    await app[BC].close()
```
i think this is meant to just refer to the variable in this file, not a member of `AppKeys`, right?

Reply:
Good catch